src/api.ts

```diff
@@ -115,38 +135,50 @@ class BigQueryAPIClient implements BigQueryAPI {
     return this.lastValidation!;
   };
 
-  getColumns = async (location: string, dataset: string, table: string, isOrderable?: boolean): Promise<string[]> => {
-    return this.fromCache('columns', this._getColumns)(location, dataset, table, isOrderable);
+  getColumns = async (
+    location: string,
+    dataset: string,
+    table: string,
+    project: string,
+    isOrderable?: boolean
+  ): Promise<string[]> => {
```

Comment:
Nit, to follow the hierarchy of BQ and other methods:

```suggestion
  getColumns = async (
    location: string,
    project: string,
    dataset: string,
    table: string,
    isOrderable?: boolean
  ): Promise<string[]> => {
```